packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -62,6 +62,11 @@ export default function CollabSidebar() {
 		};
 	}, [] );
 
+	const postStatus = useSelect((select) => {
+		const post = select(editorStore).getCurrentPost();
+		return { postStatus: post?.status };
+	}, []);
```

Comment:
This will fail ESLint checks. I would recommend using ELSlint and Prettier code editor extensions.

See: https://developer.wordpress.org/block-editor/contributors/code/getting-started-with-code-contribution/#developer-tools